decidim-comments/app/frontend/comments/add_comment_form.component.tsx

```diff
@@ -419,26 +417,28 @@ const AddCommentFormWithMutation = graphql<addCommentMutation, AddCommentFormPro
               };
             };
 
-            if (type === "Decidim::Comments::Comment") {
-                comments = prev.commentable.comments.map(commentReducer);
-              } else {
-                comments = [
-                  ...prev.commentable.comments,
-                  newComment,
-                ];
-              }
-
-            store.writeQuery({
-              query: getCommentsQuery,
-              data: {
-                ...prev,
-                commentable: {
-                  ...prev.commentable,
-                  comments,
+            if (prev) {
```

Comment:
Ts started failing after the upgrade saying that this variable could unexpectedly be `nil` sometimes. So I checked for that.